The poppler convenience library is C++ but the binding code is C.

--- origsrc/epdf/src/lib/Makefile.am	2012-09-11 11:15:40.000000000 -0500
+++ src/epdf/src/lib/Makefile.am	2012-09-11 11:31:41.602321700 -0500
@@ -42,7 +42,7 @@ libepdf_la_LIBADD = mupdf/libepdf_mupdf.
 
 else
 
-libepdf_la_LIBADD = poppler/libepdf_poppler.la
+libepdf_la_LIBADD = poppler/libepdf_poppler.la -lstdc++
 
 endif
 
